docs/resources/scenarios/analyze-web-downloads.md

```diff
@@ -24,6 +24,9 @@ You'll develop a script that analyzes weekly downloads data in the active worksh
 
 This sample was demoed as part of the Office Add-ins developer community call for February 2020.
 
+> [!NOTE]
+> The code shown in this video uses an older API model. The sample presented on this page has been updated, but the code looks a little different from the recording. The changes don't affect the behavior of the script or the other content in the presenter's demo.
```

Comment:
I think I would call out that the code shown uses the async model - just so people have context of what to look for in the code. Because you call out the code in the video, I immediately looked at it to see what the difference was. Also, you could put this note after the video, so that if people are confused they have an answer, but it doesn't prime people to look for changes right away (or be nervous about it, despite you telling them not to worry)